app/src/components/fields/dynamicFieldsSection/constants.js

```diff
@@ -23,6 +23,8 @@ export const AUTOCOMPLETE_TYPE = 'autocomplete';
 export const MULTIPLE_AUTOCOMPLETE_TYPE = 'multiAutocomplete';
 export const CREATABLE_MULTIPLE_AUTOCOMPLETE_TYPE = 'creatableMultiAutocomplete';
 
+export const ASSIGNEE = 'assignee';
```

Comment:
Let's be a bit more descriptive here
```suggestion
export const ASSIGNEE_FIELD_NAME = 'assignee';
```